controllers/all.controllers.js

```diff
@@ -26,3 +31,13 @@ exports.getArticleByID = (req, res, next) => {
       res.status(400).send({ msg: "invalid article ID" }); //
     });
 };
+
+exports.getAllArticles = (req, res) => {
+  // get the data print it
+  fetchAllArticles().then((allArticles) => {
+    console.log(allArticles, "<---- all articles from the controller");
+  });
+
+  // send the status code, along with the
+  res.status(200).send({ msg: "message from getAllArticles" });
```

Comment:
We should be sending `'allArticles'` here which has come back from the model, at the moment you're just sending a message.